### PR TITLE
update plutus

### PR DESCRIPTION
**Pull request by JaredCorduan**
This brings more expressive errors to cost model parameter validation.

### CODE REVIEW

File: eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Scripts.hs [hunk 1]
```diff
@@ -35,13 +35,15 @@ module Cardano.Ledger.Alonzo.Scripts
     ExUnits',
     Prices (..),
     hashCostModel,
-    isCostModelParamsWellFormed,
+    assertWellFormedCostModelParams,
     decodeCostModelMap,
     decodeCostModel,
     CostModels (..),
+    CostModelApplyError (..),
```
Comment (JaredCorduan) [inline eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Scripts.hs:42]: this re-export is nice, yea?

Comment (lehins) [reply eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Scripts.hs:42]: Totally!

**Review by lehins — APPROVED** [review]
:+1:

File: eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Scripts.hs (hunk 1 above)
Comment (JaredCorduan) [reply eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Scripts.hs:42]: great, thanks for confirming :)

**Pull request merged**